app/scripts/modules/core/src/application/config/trafficGuard/trafficGuardConfig.component.ts

```diff
@@ -48,13 +50,28 @@ export class TrafficGuardConfigController {
     this.viewState.originalStringVal = toJson(this.viewState.originalConfig);
 
     this.accountService.getCredentialsKeyedByAccount().then((aggregated: IAggregatedAccounts) => {
-      this.accounts = Object.keys(aggregated)
-        .map((name: string) => aggregated[name])
-        .filter((details: IAccountDetails) => details.regions);
-      this.regionsByAccount = {};
-      this.accounts.forEach((details: IAccountDetails) => {
-        this.regionsByAccount[details.name] = ['*'].concat(details.regions.map((region: IRegion) => region.name));
+      const allAccounts = Object.keys(aggregated)
+        .map((name: string) => aggregated[name]);
+      const accountsWithRegionsOnly = allAccounts
+        .filter((details: IAccountDetails) => details.regions && !details.namespaces);
+      const accountsWithNamespacesOnly = allAccounts
+        .filter((details: IAccountDetails) => details.namespaces && !details.regions);
+
+      const unsupportedAccounts = allAccounts.filter((details: IAccountDetails) => details.regions && details.namespaces);
+      if (unsupportedAccounts) {
+        this.$log.warn('Accounts ', unsupportedAccounts, ' have both namespaces and regions - this is not supported.');
```

Comment:
Don't you get the benefit of being able to expand the list in the dev console if it's not being cast to a string?